x-pack/plugins/security_solution/public/common/containers/use_full_screen/index.tsx

```diff
@@ -29,6 +29,25 @@ export const resetScroll = () => {
   }, 0);
 };
 
+export const setDataGridFullScreen = (isExpanded: boolean) => {
+  const clickEvent = new KeyboardEvent('click', {
+    view: window,
+    bubbles: true,
+    cancelable: true,
+  });
+  const dataGridIsFullScreen = document.querySelector('.euiDataGrid--fullScreen') !== null;
+  const dataGridFullScreenButtonNew = document.querySelector(
```

Comment:
my hacky sense love it, however my gray hair are telling me to not go for it and just ask EUI to create a callback on the full screen action. I really appreciate the ingenuity here.

I am sorry if I disappoint you but that a safer bet.